apps/scalable/meson.build

```diff
@@ -231,13 +232,16 @@ regular_files = [
     'xsane.svg',
     'yandex-browser.svg',
     'yuzu.svg',
-    'zotero.svg',
```

Comment:
don't change the zotero. the default icon of zotero does not scale up to gnome's HIG